cloud_testing/kotlin_poc/src/main/kotlin/ftl/ios/IosCatalog.kt

```diff
@@ -36,4 +39,9 @@ If this project has already been granted access, please email ftl-ios-feedback@g
         }
         return versionId
     }
+
+    fun supported(modelId: String, versionId: String): Boolean {
```

Comment:
I wasn't sure if we wanted a Boolean result, or to throw, like the above methods. If we do want a throw, what should be returned?

Reply:
Boolean is easier to test. We do need to throw an exception to alert the user if they've specified an invalid device though. Maybe that exception should be generated outside of IosCatalog.kt

Reply:
Probably YamlConfig should validate after parsing. What do you think?

There's a ticket to add configurable device support on iOS. That seems like a natural location for throwing the exception. #193

Reply:
Sounds good. Like in one of IosTestRunner, YamlConfig, or GcIosTestMatrix. When we go to use the values we validate and throw if necessary.

Reply:
oh yep. YamlConfig as part of #193 sounds right.